examples/templateservicebroker/templateservicebroker-template.yaml

```diff
@@ -5,12 +5,16 @@ metadata:
 parameters:
 - name: IMAGE
   value: openshift/origin:latest
-- name: TSB_TEMPLATE_NAMESPACE
-  value: openshift
 - name: NAMESPACE
   value: openshift-template-service-broker
 - name: KUBE_SYSTEM
   value: kube-system
+- name: API_SERVER_CONFIG
+  value: |
+   kind: TemplateServiceBrokerConfig
+   apiVersion: config.templateservicebroker.openshift.io/v1
+   templateNamespaces:
+   - openshift
```

Comment:
seems like this should be a parameter too, though it'll be hard to parameterize with an array value, but not impossible:
https://docs.openshift.org/latest/dev_guide/templates.html#writing-parameters